### PR TITLE
Change badge for CI to GitHub Actions.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Displays the status of CI performed in GitHub Actions.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Computer Vision Annotation Tool (CVAT)
 
-[![Build Status](https://travis-ci.org/openvinotoolkit/cvat.svg?branch=develop)](https://travis-ci.org/openvinotoolkit/cvat)
+[![CI](https://github.com/openvinotoolkit/cvat/workflows/CI/badge.svg?branch=develop&event=push)](https://github.com/openvinotoolkit/cvat/actions)
```
Comment (bsekachev) [inline README.md:3]: Do we really need ``&event=push``?

Comment (dvkruchinin) [reply README.md:3]: In this case, nightly will not be counted. Need to support nightly display too?

Comment (bsekachev) [reply README.md:3]: I do not see any reasons to ignore nightly builds. Do you see?

Comment (dvkruchinin) [reply README.md]: Thought to just separate nightly from the display. There are no reasons for this in general. Removed this filter.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**